plugins/src/messages.rs

```diff
@@ -63,7 +63,7 @@ pub struct GetManifestCall {}
 #[derive(Deserialize, Debug)]
 pub(crate) struct InitCall {
     pub(crate) options: HashMap<String, Value>,
```

Comment:
```suggestion
    pub options: HashMap<String, Value>,
```

We should make pub also this? or there is some API like `get_opt`

Reply:
Not sure, since we have accessors that return typed option values: https://github.com/ElementsProject/lightning/blob/master/plugins/src/lib.rs#L399

Then again, people might want to iterate over all options, so maybe there's no harm in providing a redundant way to access them?

Reply:
>Then again, people might want to iterate over all options, so maybe there's no harm in providing a redundant way to access them?

I do not have any strong feeling about it, let give a run period and if someone need it we make it public?